opendbc/car/tesla/carcontroller.py

```diff
@@ -6,7 +6,7 @@
 from opendbc.car.tesla.teslacan import TeslaCAN
 from opendbc.car.tesla.values import CarControllerParams
 from opendbc.car.vehicle_model import VehicleModel
-from opendbc.sunnypilot.car.tesla.mads import MadsCarController
+from opendbc.sunnypilot.car.tesla.coopsteering import CoopSteeringCarController
```

Comment:
If we do decide to change the file name, ensure to update here:
```suggestion
from opendbc.sunnypilot.car.tesla.coop_steering import CoopSteeringCarController
```